docs/src/examples/components/Button/Types/ButtonExample.tsx

```diff
@@ -1,6 +1,11 @@
 import React from 'react'
-import { Button } from '@stardust-ui/react'
+import { Button, Icon, Text } from '@stardust-ui/react'
```

Comment:
Can you please remove the unused imports.

Reply:
done, good catch! we need a tslint rule for this